docs/getting-started/new_project_guide.md

```diff
@@ -117,6 +118,16 @@ homepage]({{ site.baseurl }}/furthur-reading/clusterfuzz#web-interface).
 
 `sanitizers` example: [boringssl](https://github.com/google/oss-fuzz/blob/master/projects/boringssl/project.yaml).
 
+### architectures (optional) {#architectures}
+The list of architectures to fuzz on.
+ClusterFuzz supports fuzzing on x86_64 (aka x64) by default. However you can also fuzz using AddressSanitizer and libFuzzer on i386 (aka x86, or 32 bit) by specifying "x86_64" and "i386" in "architectures".
+This is not enabled by default because many projects won't build for i386 without some modification to their OSS-Fuzz build process.
```

Comment:
do we need to mention about instrumentation of system libraries.

Reply:
I'm not sure what you mean by this. The only difference regarding system libraries WRT i386 builds is you need to install the 32 bit ones to do it.

Reply:
ah this is asan only, so don't need to instrument them with sanitizer. i think we are not supporting msan+i386, so can skip this.